teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml

```diff
@@ -70,3 +70,6 @@
           !include-raw:
             - ../../../scripts/build_utils.sh
             - ../../build/build
+
+    publishers:
+      - github-notifier
```

Comment:
This is already taken care of by `github-pull-request`. None of the current jobs use this because it causes a conflict since it is two different plugins.

Are you seeing a problem with the current approach?